<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VotingWindow</class>
 <widget class="QMainWindow" name="VotingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777214</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_status">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="text_theme_search">
      <property name="placeholderText">
       <string>Search for Themes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="list_themes"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_vote_yes">
        <property name="text">
         <string>Vote YES</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_vote_no">
        <property name="text">
         <string>Vote NO</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_flag">
        <property name="text">
         <string>FLAG</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_pyJAMa">
   <property name="text">
    <string>About pyJAMa</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionInput_token">
   <property name="text">
    <string>Input Token</string>
   </property>
  </action>
  <action name="actionTest_Connection">
   <property name="text">
    <string>Test Connection</string>
   </property>
  </action>
  <action name="actionReload_all_Data">
   <property name="text">
    <string>Reload all Data</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
